src/components/Form/__tests__/Label.jsx

```diff
@@ -15,8 +15,7 @@ describe('<Form.Label />', () => {
   })
 
   it('takes attributes', () => {
-    const $ = shallow(<Form.TextInput placeholder="test" id="foo" htmlFor="t" />)
-    expect($.prop('placeholder')).toBe('test')
+    const $ = shallow(<Form.Label id="foo" htmlFor="t" />)
```

Comment:
This test still passes despite `id` in `Label` being hard coded as the `...rest` spread in `Label` overrides it.  You can just remove the `id` prop from `Label`